pkg/http/server.go

```diff
@@ -155,8 +155,18 @@ func NewServer(addrs []string, handler http.Handler, certificate *tls.Certificat
 	if certificate != nil {
 		tlsConfig = &tls.Config{
 			PreferServerCipherSuites: true,
-			MinVersion:               tls.VersionTLS12,
-			NextProtos:               []string{"http/1.1", "h2"},
+			CipherSuites: []uint16{
```

Comment:
We should take this list as perhaps an argument to NewServer() function, in future we might need to change these as well.  We can take this as an input from top level and not default/restrict this package to the following ciphers.

Reply:
Agree 👍